src/workflow/reboot.c

```diff
@@ -39,9 +39,12 @@ static void _reboot(void)
 
 bool workflow_reboot(void)
 {
+#if PLATFORM_BITBOX02 == 1
```

Comment:
maybe slightly more future-robust if you checked for the base platform instead, as this could misbehave if we add another platform in the future :sweat_smile: